index.css

```diff
@@ -30,15 +30,15 @@ button:hover {
 .cat {
   width: 100px;
   height: 100px;
-  margin-left: 100px;
+  margin-left: 20px;
```

Comment:
Moved the cat over so it wouldn't slide 'off screen' when code/preview are both open in embed